lib/makeup_sql.ex

```diff
@@ -1010,9 +1007,19 @@ defmodule MakeupSql do
     choice((built_in ++ Enum.map(built_in, &String.downcase/1)) |> Enum.map(&string/1))
     |> token(:keyword_type)
 
-  whitespace = ascii_char([?\s, ?\t, ?\r, ?\n, ?\f]) |> token(:whitespace)
+  @known_whitespace_characters [?\r, ?\s, ?\n, ?\f, ?\t]
+
+  whitespace = ascii_string(@known_whitespace_characters, min: 1) |> token(:whitespace)
   newline = ascii_char([?\r, ?\n])
 
+  keyword_constants =
+    word_from_list(
+      ["TRUE", "FALSE", "NULL", "true", "false", "null"],
+      :keyword_constant
+    )
+
+  any_char = utf8_char([]) |> token(:error)
```

Comment:
It's from makeup source code

Reply:
Hey @Billzabob . Do you plan to maintain this package ?

Reply:
Oh wow, totally forgot about this. My bad. Ya I don’t mind maintaining it. Seems like a pretty low maintenance library anyways. I’ll get this merged and published for you tomorrow 👍

Reply:
Published! Thanks for the reminder and sorry for taking a year to get this out 😅